src/aks-preview/azext_aks_preview/__init__.py

```diff
@@ -16,7 +16,7 @@ class ContainerServiceCommandsLoader(AzCommandsLoader):
 
     def __init__(self, cli_ctx=None):
         from azure.cli.core.commands import CliCommandType
-        register_resource_type('latest', CUSTOM_MGMT_AKS_PREVIEW, '2020-02-01')
+        register_resource_type('latest', CUSTOM_MGMT_AKS_PREVIEW, '2020-03-01')
```

Comment:
any breaking change in the new api version, just to confirm cli extension version bumping following semver.